src/main/java/frc/robot/subsystems/conveyor/commands/LoadConveyor.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+/**
+ * This command load the conveyor, as long as the conveyor isn't "out of space".
+ */
+public class LoadConveyor extends CommandBase {
+    private final Conveyor conveyor;
+    private final double power;
+
+    public LoadConveyor(Conveyor conveyor, double power) {
+        this.conveyor = conveyor;
+        this.power = power;
+
+        addRequirements(conveyor);
+    }
+
+    @Override
+    public void execute() {
+        if (!Conveyor.isConveyorFull()) {
```

Comment:
Maybe use the funnel sensor to check if the ball entered

Reply:
How would it help in a case we are planning to load more than one ball?

Reply:
You would move the conveyor while there is a ball at the entrence